GOVERNANCE.md

```diff
@@ -0,0 +1,150 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the
+`zarr` project, to clarify how decisions are made and how the various
+elements of our community interact.
+
+This is a consensus-based community project. Anyone with an interest in the
+project can join the community, contribute to the project design, and
+participate in the decision making process. This document describes how that
+participation takes place, how to find consensus, and how deadlocks are
+resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The zarr community consists of anyone using or working with the project
+in any way.
+
+## Contributors
+
+A community member can become a contributor by interacting directly with the
+project in concrete ways, such as:
+
+- proposing, discussing, or reviewing a change to the code, documentation, or specification
+  via a GitHub pull request to any https://github.com/zarr-developers repository;
+- reporting a GitHub issue
+  to any https://github.com/zarr-developers repository;
+- discussing examples or usage issues under the #zarr tag on StackOverflow 
+  (like "How do I do X in zarr?").
+
+among other possibilities. Any community member can become a contributor, and
+all are encouraged to do so. By contributing to the project, community members
+can directly help to shape its future.
+
+Contributors are encouraged to read the [contributing guide](http://zarr.readthedocs.io/en/stable/contributing.html)
+for information about contributing to Zarr.
+
+## Core developers
+
+Core developers are community members that have demonstrated continued
+commitment to the project through ongoing contributions. They
+have shown they can be trusted to maintain zarr with care. Becoming a
+core developer allows contributors to merge approved pull requests, cast votes
+for and against merging a pull-request, and be involved in deciding major
+changes to the API, and thereby more easily carry on with their project related
+activities. Core developers appear as organization members on the zarr
+[GitHub organization](https://github.com/orgs/zarr-developers/people) and are on our
+[@zarr-developers/core-devs](https://github.com/orgs/zarr-developers/teams/core-devs) GitHub team. Core
```

Comment:
I don't think this link is publicly accessible. It appears you need to (1) have a GitHub account and (2) either be a member of the `zarr-developers` org or the `core-devs` team to view the page.

We could consider removing the `core-devs` team link and just stick with the `zarr-developers` org members. Or if we're feeling more ambitious we could add additional content to https://zarr-developers.github.io which lists core developers. Perhaps using something along the lines of https://jupyter.org/about as a model

Reply:
Having a page sounds good. Someone want to give it a shot?